contracts/MultiSigWallet.sol

```diff
@@ -590,21 +587,27 @@ contract MultiSigWallet {
      * @dev Transaction is confirmed if wallets' count that confirmed
      *      the transaction is bigger or equal to required.
      *      Function checks confirmation condition based on current set of
-     *      registered wallet.
+     *      registered wallets.
+     *      Function returns true if transaction was executed despite if
+     *      with current set of registered wallets it might not.
      *      Function requires:
      *          - Transaction with the specified id exists.
      *
      * @param _transactionID Transaction id to check.
      *
-     * @return Returns true in case if transaction confirmed, otherwise
-     *         false.
+     * @return Returns true if the transaction is executed or confirmation is
+     *         achieved with current set of registered wallets, otherwise false.
      */
     function isTransactionConfirmed(uint256 _transactionID)
         internal
         view
         transactionExists(_transactionID)
         returns (bool)
     {
+        if (transactions[_transactionID].executed) {
+            return true;
+        }
```

Comment:
With above change method can be made public now. Caller can call this method to check if transaction is confirmed or not. Irrespective of transaction is executed or not,  isTransactionConfirmed will work properly.

Reply:
We can have for now it public.